karyon3-log4j2/src/main/java/org/apache/logging/log4j/core/ArchaiusLog4J2ConfigurationFactory.java

```diff
@@ -54,6 +58,13 @@ public static void initialize(Config _config) {
         ctx.reconfigure();
     }
     
+    @Inject()
+    public static void initialize(@Nullable Set<Log4jConfiguration> _appenders) {
```

Comment:
Combine this with the initialize above to make

initialize(Config _config, Set<Log4jConfiguration> _configs)